src/System.Dynamic.Runtime/tests/Dynamic.Statements/Conformance.dynamic.statements.IndexerOperator.Unary.cs

```diff
@@ -34,7 +34,7 @@ namespace ManagedTests.DynamicCSharp.Conformance.dynamic.statements.IndexerOpera
     using ManagedTests.DynamicCSharp.Conformance.dynamic.statements.IndexerOperator.Unary.decrease001.decrease001;
     // <Area>operator on dynamic indexer</Area>
     // <Title> unary operator  </Title>
-    // <Description>prefix increasement</Description>
```

Comment:
Depending on how one defines "real word", it is. It's attested, and goes back to Middle English which should satisfy purists that it's "real" English. It's rare enough to be worth avoiding though.